editor/src/core/model/element-metadata-utils.ts

```diff
@@ -2155,6 +2159,36 @@ function fillSpyOnlyMetadata(
   return workingElements
 }
 
+function fillMissingDataFromAncestors(mergedMetadata: ElementInstanceMetadataMap) {
+  let workingElements: ElementInstanceMetadataMap = { ...mergedMetadata }
+
+  const elementsWithoutGlobalContentBox = Object.keys(workingElements).filter((p) => {
```

Comment:
Minor point, but you could use [`Object.entries`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) here instead